### PR TITLE
[10.1.x] ISPN-11825 update procedure to register schemas with CLI

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11825 - for 10.1.x only. Will address changes for 11 separately.

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/query.adoc [hunk 1]
```diff
@@ -1489,26 +1489,63 @@ marshaller. This is _key_ for the search capability to work. But it's also possi
...
+
+==== Registering Protobuf Schemas on {brandname} Servers
+{brandname} servers need to access indexing metadata from the same descriptor,
+`.proto` file, as clients. For this reason, {brandname} servers store `.proto`
+files in a dedicated cache, `___protobuf_metadata`, that stores both keys and
+values as plain strings.
+
+.Prerequisites
+
+* If you use cache authorization to control access, assign users the
```
Comment (anistor) [inline documentation/src/main/asciidoc/topics/query.adoc:1512]: this is only needed for write access. might want to add 2 words here to make it clear

File: documentation/src/main/asciidoc/topics/query.adoc [hunk 2]
```diff
@@ -1489,26 +1489,63 @@ marshaller. This is _key_ for the search capability to work. But it's also possi
...
+pass:['___schema_manager'] role to access the `___protobuf_metadata` cache.
+
+.Procedure
+
+To register a schema with {brandname} server, perform a _put_ operation on the
```
Comment (anistor) [inline documentation/src/main/asciidoc/topics/query.adoc:1517]: the PUT operation is hidden behind the ```schema``` command.

File: documentation/src/main/asciidoc/topics/query.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/query.adoc:1512]: thanks Adrian

File: documentation/src/main/asciidoc/topics/query.adoc (hunk 2 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/query.adoc]: yeah, good point. we can trim some of the fat here.

**Comment by anistor** [comment]
LGTM

**Comment by anistor** [comment]
Integrared. Thanks  @oraNod !

**Pull request merged**